code/game/gamemodes/game_mode.dm

```diff
@@ -230,7 +230,11 @@ var/global/list/additional_antag_types = list()
 	refresh_event_modifiers()
 
 	spawn (ROUNDSTART_LOGOUT_REPORT_TIME)
-		display_roundstart_logout_report()
+		var/logout_report = get_logout_report()

```

Comment:
Proposal, leave the proc `display_roundstart_logout_report()` and just have it call `get_logout_report()` within it. Should we ever want to display the report to all staff somewhere else, it can be re-used easily enough. Also monolith functions.